appformer-js-monaco/pom.xml

```
@@ -22,6 +22,9 @@
         <artifactId>frontend-maven-plugin</artifactId>
         <configuration>
           <installDirectory>${project.build.directory}</installDirectory>
+          <nodeVersion>v16.2.0</nodeVersion>
+            <npmVersion>7.15.1</npmVersion>
+            <yarnVersion>v1.22.10</yarnVersion>
```

Hi @Ginxo, thanks for the PR!
Can you please indent these lines? 
Other than this, LGTM.

```suggestion
          <nodeVersion>v16.2.0</nodeVersion>
          <npmVersion>7.15.1</npmVersion>
          <yarnVersion>v1.22.10</yarnVersion>
```